### DISCUSSION TITLE
Cross compiling on Linux for Windows with QT backend

### DISCUSSION THREAD

**Discussion by DBLouis** [slint-ui/slint#1444] [General]
Hi
Is it supported?
I tried to build for windows with only the QT backend enabled. I use the mingw toolchain (target = `x86_64-pc-windows-gnu`) and I get this error:

```
note: /usr/lib/gcc/x86_64-w64-mingw32/11.1.0/../../../../x86_64-w64-mingw32/bin/ld: cannot find -lQt5Core
      /usr/lib/gcc/x86_64-w64-mingw32/11.1.0/../../../../x86_64-w64-mingw32/bin/ld: cannot find -lQt5Gui
      /usr/lib/gcc/x86_64-w64-mingw32/11.1.0/../../../../x86_64-w64-mingw32/bin/ld: cannot find -lQt5Widgets
      collect2: error: ld returned 1 exit status
```

Do I need to build those libs myself?

**Comment by tronical** [comment]
Interesting, I’ve never tried this and I unfortunately don’t have experience with this particular combination of cross-compilation. That said, it looks like you’re pretty close - after all this is the linking stage.

To answer your concrete question: Yes, you’ll definitely need the Qt DLLs for Windows available for the linker to find them.
